tests/phpunit/CommonExtractorTest.php

```diff
@@ -338,98 +338,55 @@ public function testGetTablesAction(): void
             0 =>
                 [
                     'name' => 'escaping',
-                    'sanitizedName' => 'escaping',
                     'schema' => 'testdb',
-                    'type' => 'BASE TABLE',
                     'columns' =>
                         [
-                            0 =>
-                                [
-                                    'name' => 'col1',
-                                    'sanitizedName' => 'col1',
-                                    'type' => 'varchar',
-                                    'primaryKey' => false,
-                                    'length' => '155',
-                                    'nullable' => false,
-                                    'default' => 'abc',
-                                    'ordinalPosition' => '1',
-                                ],
-                            1 =>
-                                [
-                                    'name' => 'col2',
-                                    'sanitizedName' => 'col2',
-                                    'type' => 'varchar',
-                                    'primaryKey' => false,
-                                    'length' => '155',
-                                    'nullable' => false,
-                                    'default' => 'abc',
-                                    'ordinalPosition' => '2',
-                                ],
+                            [
+                                'name' => 'col1',
```

Comment:
Tento test je upraveny, ... lebo vo vysledku getTables posielali zbytocne kluce, ... ktore tam UI neocakava ... konzultovane s JKotek: https://keboola.slack.com/archives/CMTBFNLF8/p1588684823012800

Bolo to z dovodu, ze sa rovneke pole pouzivalo pre generovanie metadatat do manifestu aj do getTables vystupu.

Reply:
myslim si, ze je to spravne ... kedze "serializacia" metadata je uz rozdelena a uz nie je spojene generovanie manifestu a getTables vystupu ... v metadata objektoch mas teraz vsetky metadata .... a `GetTablesSerializer` z toho vezme co je potrebne pre getTables vystup ..... `ManifestSerializer` z toho vezme co je potrebne pre manifest.

**Dokazuju to aj testy:**
- test, ktory tu komentujes sa z uvedenych dovodov zmenil
- no test na manifest zostal bez zmeny (nie je ani tu v diffe): https://github.com/keboola/db-extractor-common/blob/8adef40c581d3a76cfbf9095949f82271e6f96a6/tests/phpunit/CommonExtractorTest.php#L397-L545

Su tam na to aj osobitne metody:
- `BaseExtractor::getManifestMetadataSerializer` 
- `BaseExtractor::getGetTablesMetadataSerializer`

GetTables sa potom serializuje v `BaseExtractor::getTables()`.

Manifest sa serializuje v `BaseExtractor::createManifest()`.

@ondrajodas  Dava to takto zmysel, alebo to mam este nejak viac upresniut?

Reply:
jo jasně, už to chápu 👍 díky za upřesnění :)